toolkit/tools/imagegen/attendedinstaller/speakuputils/speakuputils.go

```diff
@@ -18,7 +18,7 @@ const (
 
 // CreateVirtualKeyboard creates and returns a virtual keyboard from the uinput package
 func CreateVirtualKeyboard() (keyboard uinput.Keyboard, err error) {
-	keyboard, err = uinput.CreateKeyboard("/dev/uinput", []byte("MarinerVirtualKeyboard"))
+	keyboard, err = uinput.CreateKeyboard("/dev/uinput", []byte("AzureLinuxVirtualKeyboard"))
```

Comment:
consider changing to "myLinuxVirtualKeyboard" or "isoVirtualKeyboard"

Reply:
Updated